projects/flow.groups-2D-random-walk/src/project.py

```diff
@@ -0,0 +1,152 @@
+import flow
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+# Helper functions
+def get_first_replica(jobs, first_index=1):
+    try:
+        job = next(filter(lambda job: job.sp.replica == first_index, jobs))
+    except StopIteration:
+        raise RuntimeError("First replica not found.")
+    return job
+
+
+def positions_generator(jobs):
+    for job in jobs:
+        with job.data:
+            yield job.data.positions
+
+
+def aggregate_cond_true(doc_attr):
```

Comment:
I think this is exactly the same as the default behavior of the `true` condition with aggregation enabled.